xcode/Subconscious/Shared/Library/CollectionUtilities.swift

```diff
@@ -36,3 +36,9 @@ extension Array {
         return unique
     }
 }
+
+extension Array where Element: Hashable {
+    func uniquing() -> [Element] {
+        uniquing(with: { element in element })
+    }
+}
```

Comment:
Add synonym for uniquing that does not require a callback when element is hashable.